### PR TITLE
Add ProbeControlRoom

### PR DESCRIPTION
**Pull request by icedown**

### CODE REVIEW

File: NetKAN/ProbeControlRoom.netkan [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version": "1.2.1.0",
+    "identifier"  : "ProbeControlRoom",
+    "license"     : "CC-BY-NC-SA-3.0",
+	"$vref" 	  : "#/ckan/ksp-avc",
+	"#kref"		  : "#/ckan/spacedock/1061",
+	"download"	  : "https://github.com/icedown/KSP-ProbeControlRoom/releases/download/1.2.1.0/ProbeControlRoom1.2.1.0.zip",
```
Comment (politas) [inline NetKAN/ProbeControlRoom.netkan]: $kref inflation should provide the required download, please delete this line.

File: NetKAN/ProbeControlRoom.netkan [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version": "1.2.1.0",
```
Comment (politas) [inline NetKAN/ProbeControlRoom.netkan]: "spec_version" refers to the CKAN spec version, not the mod version. This should be:

    "spec_version": 1,

File: NetKAN/ProbeControlRoom.netkan [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+		{ "name"  : "RasterPropMonitor"}
+	],
+	"recommends"  : [
+	{
+		"name"	  : "Toolbar"
+	}],
+	"suggests"    : [
+		{ "name"  : "ScanSat"},
+		{ "name"  : "VesselViewer"},
+		{ "name"  : "MechJeb"},
```
Comment (Dazpoet) [inline NetKAN/ProbeControlRoom.netkan]: This should be `MechJeb2` since that is our identifier for it

File: NetKAN/ProbeControlRoom.netkan [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	"depends" 	  : [
+		{ "name"  : "ModuleManager" },
+		{ "name"  : "RasterPropMonitor"}
+	],
+	"recommends"  : [
+	{
+		"name"	  : "Toolbar"
+	}],
+	"suggests"    : [
+		{ "name"  : "ScanSat"},
```
Comment (Dazpoet) [inline NetKAN/ProbeControlRoom.netkan]: This should be SCANsat to match the identifer used in our repositories.

File: NetKAN/ProbeControlRoom.netkan [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version": "1",
```
Comment (Dazpoet) [inline NetKAN/ProbeControlRoom.netkan]: There should be no quotation marks around the 1, it's a special case due to arcane reasons long forgotten

**Comment by politas** [comment]
Only thing holding it back now appears to be the dependency on RasterPropMonitor, which is not yet available for KSP 1.2.x

**Comment by linuxgurugamer** [comment]
#rebuild

**Comment by linuxgurugamer** [comment]
Removing the BlockedOnDependency since RPM is available and it passed the checks

**Comment by politas** [comment]
Cool!

**Pull request merged**